java/ql/integration-tests/all-platforms/java/diagnostics/no-gradle-wrapper/diagnostics.expected

```diff
@@ -0,0 +1,14 @@
+{
+  "markdownMessage": "Built a Gradle project without the [https://docs.gradle.org/current/userguide/gradle_wrapper.html](Gradle wrapper). This may use an incompatible version of Gradle",
```

Comment:
```suggestion
  "markdownMessage": "Built a Gradle project without the [Gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html). This may use an incompatible version of Gradle",
```